lib/install/data/data.go

```diff
@@ -216,6 +220,39 @@ func (d *Data) copyContainerNetworks(src *Data) error {
 	return nil
 }
 
+// copyVolumeStores checks that existing volume stores (in d) are present in the
+// specified volume stores (src) and then copies new volume stores into d. It
+// does not overwrite data for existing volume stores.
+func (d *Data) copyVolumeStores(src *Data) error {
+	// Any existing volume stores must be specified while performing a configure operation.
+	errMsg := "Existing volume store %s not specified in configure command"
+	for label, url := range d.VolumeLocations {
+		srcURL, ok := src.VolumeLocations[label]
+		if !ok {
+			log.Errorf(fmt.Sprintf(errMsg, label))
+			log.Info(fmt.Sprintf(copyInfoMsg, "volume store", "volume stores"))
```

Comment:
suggest to finish loop here if any existing volume store is not specified, so user can get all missing configurations at one time